UT4MasterServer/appsettings.Development.json

```diff
@@ -1,4 +1,9 @@
 {
+	"ApplicationSettings": {
+		"WebsiteScheme": "http",
+		"WebsiteDomain": "localhost",
+		"WebsitePort": 8080
```

Comment:
Add a check to `Program.cs` to ensure that `WebsiteScheme` and `WebsiteDomain` are not empty.
In development environment they may default to these values when not set (including `WebsitePort`).

Reply:
Added checks.